tests/load/pipeline/test_merge_disposition.py

```diff
@@ -139,7 +139,7 @@ def test_merge_on_ad_hoc_primary_key(destination_config: DestinationTestConfigur
 
 
 @dlt.source(root_key=True)
-def github():
+def github(remove_lists: bool = False):
```

Comment:
see above, for clickhouse i need to remove the lists in the dataset, otherwise it will look for a List[json] field and not find it. needs to go into the docs

Reply:
removed this, no longer needed for disabled complex types